package.json

```diff
@@ -74,8 +74,8 @@
     "universal-cookie": "4.0.4"
   },
   "peerDependencies": {
-    "@edx/paragon": ">= 10.0.0 < 21.0.0",
     "@edx/frontend-build": ">= 8.1.0",
```

Comment:
Will need to bump peer dependency of `@edx/frontend-build` once https://github.com/openedx/frontend-build/pull/365 merges.